packages/lib/event-types/transformers/api-request.ts

```diff
@@ -162,6 +164,17 @@ function transformApiEventTypeFutureBookingLimits(
   }
 }
 
+function transformApiEventTypeRecurrence(
+  recurrence: CreateEventTypeInput_2024_06_14["recurrence"]
+): TransformRecurringEventSchema_2024_06_14 | undefined {
+  if (!recurrence) return undefined;
+  return {
+    interval: recurrence.interval,
+    count: recurrence.occurrences,
+    freq: Frequency[recurrence.frequency as unknown as keyof typeof Frequency],
+  } as TransformRecurringEventSchema_2024_06_14;
```

Comment:
satisfies TransformRecurringEventSchema_2024_06_14

Reply:
made changes in https://github.com/calcom/cal.com/pull/16251/commits/126a7b7cbffc50825b3e4ada442db69221d4edd1

Reply:
Fixed